build/openwisp_dashboard/urls.py

```diff
@@ -1,22 +1,46 @@
+from openwisp_network_topology.visualizer import urls as visualizer_urls
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.urls import reverse_lazy
 from django.views.generic import RedirectView
 from django.conf import settings
 from django.conf.urls.static import static
-from openwisp.utils import openwisp_topology_urls, openwisp_controller_urls
+from .topology_urls import receive_name, urlpatterns as topology_urls
+from .views import topology_redirect_view
 
-redirect_view = RedirectView.as_view(url=reverse_lazy('admin:index'))
+
+index_redirect_view = RedirectView.as_view(url=reverse_lazy('admin:index'))
 
 urlpatterns = [
+    url(r'^$', index_redirect_view, name='index'),
     url(r'^admin/', admin.site.urls),
     url(r'^accounts/', include('openwisp_users.accounts.urls')),
-    url(r'^$', redirect_view, name='index'),
-    url(r'^', include('openwisp_radius.urls', namespace='freeradius')),
+    url(r'^topology/', include(visualizer_urls)),
 ]
 
-urlpatterns += openwisp_controller_urls()
-urlpatterns += openwisp_topology_urls()
+
+# Create a url for topology receive pattern
+def topology_url(urlpatterns, pattern, receive_name):
+    receive_url = url(pattern,
+                      topology_redirect_view,
+                      name=receive_name)
+    urlpatterns.append(receive_url)
+
+
+for api_url in topology_urls:
+    if hasattr(api_url, 'name') and api_url.name == receive_name:
```

Comment:
What do you think about this hacky solution? 
I am copying the path from `topology_urls.py` file and registering it as a redirect url to same path but topology domain:
```
def topology_redirect_view(request, pk):
    url = "{0}://{1}{2}".format(request.scheme,
                                os.environ["TOPOLOGY_DOMAIN"],
                                request.get_full_path())
```
make sure that dashboard gets the path while the url points to topology container. :smile: